src/main/java/com/icthh/xm/tmf/ms/activation/service/SagaServiceImpl.java

```diff
@@ -673,7 +678,7 @@ private void generateIterableEvent(String sagaTransactionId, String parentTypeKe
             .setIterationsCount(countOfIteration);
         sagaEvent.markAsInQueue();
         sagaEvent = sagaEventRepository.save(sagaEvent);
-        eventsManager.sendEvent(sagaEvent);
+        sendEventOnCommitted(sagaEvent);
```

Comment:
eventsManager.sendEvent is also used in `resendEventsForDependentTasks`. Please check if this `sendEventOnCommitted` wrapper should be used there